src/ui/src/app/header-bar/header-bar.component.scss

```diff
@@ -29,51 +37,112 @@ $header-bar-padding-small: 1.5em;
 
   &__search {
     margin-top: .5em;
-
-    .md-input-wrapper {
-      margin: .4em 0;
+    background: rgba(255, 255, 255, 0.125);
+    border-radius: 4px;
+    transition: background 0.2s ease;
+    padding: 0.3em 0.5em;
+    svg {
+      fill: rgba(255, 255, 255, 0.6);
```

Comment:
Is this necessary? We can update the SVG file to use the color defined in `color` property instead of force the color in the fill property.